ql/src/test/queries/clientpositive/insert_with_union_and_tez.q

```diff
@@ -0,0 +1,16 @@
+set tez.am.speculation.enabled=true;
```

Comment:
How is AM speculative execution affecting this behaviour?

Reply:
Oh sorry! I thought it's for the tasks, I will remove it and push again.